metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -560,7 +572,9 @@ def _get_chart_type(
         return chart_type
 
     def _make_chart_metadata_events(
-        self, dashboard_element: LookerDashboardElement, dashboard: LookerDashboard
+        self,
+        dashboard_element: LookerDashboardElement,
+        dashboard: Optional[LookerDashboard],
```

Comment:
add a comment saying that dashboard will be None if this is a standalone look

Reply:
done